generic_tracers/generic_COBALT.F90

```diff
@@ -4249,6 +4256,7 @@ subroutine generic_COBALT_update_from_source(tracer_list,Temp,Salt,rho_dzt,dzt,h
              ! Since burial is highly uncertain and often used in global earth system simulations to balance inputs and
              ! outputs, a dimensionless scaling factor (cobalt%scale_burial) has also been included.
              fpoc_btm = cobalt%fntot_btm(i,j)*cobalt%c_2_n*sperd*1000.0
+             ! Should we use ztop?
```

Comment:
Hmm. This depends on how we define our benthic layer. If we consider the model bottom layer as the benthic layer, then using `ztop` would be appropriate. However, if we define the benthic layer as one layer below our model's bottom grid, then `cobalt%zt` would be suitable.

Reply:
I know I have looked at this directly before, and maybe I'm just thinking myself in a circle, but am I right in thinking that grid_kmt is equal to the number of vertical tracer grid cells (e.g., 75 for most of our regional applications) in MOM6.  So "zt(75)" is the bottom of the 75th grid cell (i.e., the sediment)?  I just need a quick sanity check on this.